content-src/components/Search/Search.js

```diff
@@ -45,7 +45,7 @@ const Search = React.createClass({
     const numSuggestions = this.props.formHistory.concat(this.props.suggestions).length;
     const numEngines = this.props.engines.length;
     if (index < numSuggestions) {
-      return "search-suggestions-" + "history-search-suggestions-" + index;
+      return `${index < this.props.formHistory.length ? "history-" : ""}search-suggestions-${index}`;
```

Comment:
This seems odd to me, and a big change in logic.

The old version looks like we were returning:
- `search-suggestions-history-search-suggestions-${index}`

... but the new version seems like we're returning one of the following:

- `history-search-suggestions-${index}`
- `search-suggestions-${index}`